bigchaindb/common/schema/transaction_migration_election_v2.0.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright BigchainDB GmbH and BigchainDB contributors
+# SPDX-License-Identifier: (Apache-2.0 AND CC-BY-4.0)
+# Code is Apache-2.0 and docs are CC-BY-4.0
+
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
```

Comment:
This schema has a lot of unnecessary details. We should just validate the transaction agains the base election schema [documented here](https://github.com/bigchaindb/BEPs/pull/82/files) and additionally assert the operation is `TENDERMINT_MIGRATION_ELECTION`.

Reply:
I would suggest merging (if possible) the two yaml files i.e. the base election spec and migration election spec once they are loaded in `_load_schema` function. This we would only need one additional schema validation instead of two but we will still retain different base election spec and migration election sepc.

Reply:
I do not see the yaml file to be a necessity but does not hurt to have it either. The important bit - the only thing it should describe is custom operation.

Reply:
> I do not see the yaml file to be a necessity 
I would like to point out that the yaml files for a particular election transaction define the `asset.data` field. So imho it seems necessary

Reply:
Discussed offline - we do not strictly need a schema for the migration election. Transactions follow the base election schema described in [BEP-18](https://github.com/bigchaindb/BEPs/pull/82) and additionally have a custom `TENDERMINT_MIGRATION` operation.

Reply:
Deferred to another PR. https://github.com/bigchaindb/bigchaindb/issues/2542